packages/flutter_tools/templates/plugin/android.tmpl/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
```

Comment:
This is the only one I'm not 100% sure about whether we should touch.  This affects customers.

Reply:
Should we split it to another PR? WDYT @blasten?

Reply:
this file isn't really used. I don't think it will break customers as far as I can tell.

Reply:
Actually. This is used if a developer tries to build the plugin alone using Android Studio. cc @gaaclarke who just had this issue.